src/System.Runtime.Serialization.Xml/tests/Performance/Perf.Deserialization.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using System;
+using System.IO;
+using Microsoft.Xunit.Performance;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace System.Runtime.Tests
+{
+    public class Perf_Deserialization
+    {
+        [Benchmark]
+        public void DeserializeLargeObjectGraph()
+        {
+            var obj = CreateObject(SimpleType.GraphSize, 0);
+            var serializer = new DataContractSerializer(typeof(SimpleType));
+            using (var stream = new MemoryStream())
+            {
+                serializer.WriteObject(stream, obj);
+                stream.Position = 0;
+
+                foreach (var iteration in Benchmark.Iterations)
+                    using (iteration.StartMeasurement())
+                        for (int i = 0; i < 3; i++)
+                        {
+                            var obj2 = serializer.ReadObject(stream);
+                            stream.Position = 0;
```

Comment:
If you move this up one line, you could get rid of the earlier stream.Position = 0 call.